### ISSUE TITLE
Edit Question: [ua_ruhr/metadata-and-referencing/structure-granularity-and-referencing-pids]

### ISSUE THREAD

**Issue by jmfrenzel** [FDM-UARuhr/rdmo-catalog-uaruhr#46]
**Describe the request**  
Add missing words PID question set 

**Propose what exactly should be changed**  
Add word in help text (de) locale <questionset dc:uri="https://rdmorganiser.github.io/terms/questions/ua_ruhr/metadata-and-referencing/structure-granularity-and-referencing-pids">

**Original setting or content**   

*question help text de*  
".... Die PID bleibt gleich, auch wenn der Speicherort des Objektes sich. ...."

**Proposed setting or content**   
*question help text de*  
".... Die PID bleibt gleich, auch wenn der Speicherort des Objektes sich ändert. ...."

**Additional context**

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  

- [x] questionset  help text (ger)

**Comment by ClaraRosalie** [comment]
already implemented in [all_questions.xml](https://github.com/FDM-UARuhr/rdmo-catalog-uaruhr/edit/master/tools/all_questions.xml)
